Cargo.toml

```diff
@@ -78,7 +78,10 @@ license = "Apache-2.0"
 repository = "https://github.com/risingwavelabs/risingwave"
 
 [workspace.dependencies]
-foyer = { version = "0.10.4", features = ["nightly", "mtrace"] }
+foyer = { git = "https://github.com/foyer-rs/foyer", rev = "f8eae8c", features = [
```

Comment:
Why not publishing a version?

Reply:
Let me first build an image of this PR. If everything works fine, I'll bump foyer version, otherwise I need to fix bugs first.